torch_xla/csrc/ops/adaptive_avg_pool3d.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
```

Comment:
Needs:
```c++
#include <vector>
```

Reply:
Why? If we are missing this include it should not compile correct?

Reply:
It happens to compile correctly because one of the included files ends up including `vector`.
If somehow such included file stops including `vector` you get compile errors in unrelated parts of the code, which are very annoying,
This is why C++ code must include the proper files of the used interfaces.

Reply:
Thank you. Should I check this in other source code as well? by the same logic it is missing in pooling.h, adaptive_avg_pool2d.h, and perhaps many other files. Thoughts?

Reply:
I dont know which comments, I marked resolved. Unless, I push any change, resolution does not make sense. I trust you spotted a legitimate error I made, thanks for pointing out, I will careful next time.